README.md

```diff
@@ -148,74 +122,31 @@ Example commands:
 ```
 --->
 
-### Install
-
-Install this project with:
-
-<!--
-Example command:
-
-```sh
-  cd my-project
-  gem install
-```
---->
 
 ### Usage
 
 To run the project, execute the following command:
+the use of live server  is recommended 
 
-<!--
-Example command:
-
-```sh
-  rails server
-```
---->
-
-### Run tests
-
-To run tests, run the following command:
-
-<!--
-Example command:
-
-```sh
-  bin/rails test test/models/article_test.rb
-```
---->
 
 ### Deployment
 
 You can deploy this project using:
 
-<!--
-Example:
-
-```sh
-
-```
- -->
+not ready for deployment
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- AUTHORS -->
 
-## 👥 Authors <a name="authors"></a>
+## 👥 Authors <a name="authors">Iheagwara Augustine</a>
 
 > Mention all of the collaborators of this project.
```

Comment:
- [ ] As suggested [here](https://github.com/stino-x/HELLO-NEW/pull/2#pullrequestreview-1247915815), please consider removing this comment. It is a mere template as guide on how to fill your information in.